dev/devicelab/manifest.yaml

```diff
@@ -401,12 +401,30 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
-  integration_ui:
+  integration_ui_driver:
     description: >
       Runs end-to-end Flutter tests on Android.
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  integration_ui_keyboard_resize:
+    description: >
+      Runs end-to-end Flutter tests on Android.
```

Comment:
These duplicated descriptions are kind of lame since they are testing different things, but the name of the test probably covers it.